source/DIYLinux.rst

```diff
@@ -0,0 +1,153 @@
+.. _DIYLinux:
+
+DIY Linux  
+==========
```

Comment:
Does DIY add anything here? Would it be cleaner to combine these headings and write "Installing Ubuntu Linux"?

Reply:
agreed